quickwit/quickwit-query/Cargo.toml

```diff
@@ -18,10 +18,12 @@ lindera-core = { workspace = true, optional = true }
 lindera-dictionary = { workspace = true, optional = true }
 lindera-tokenizer = { workspace = true, optional = true }
 once_cell = { workspace = true }
+regex = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 serde_with = { workspace = true }
 tantivy = { workspace = true }
+time = { workspace = true }
```

Comment:
```suggestion
```

Reply:
This one is required by `range_query.rs`
```
use time::format_description::well_known::Rfc3339;
```